bouachain/chain.json

```diff
@@ -0,0 +1,111 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "bouachain",
+    "chain_type": "cosmos",
+    "status": "live",
+    "website": "https://bouachain.com",
+    "network_type": "mainnet",
+    "pretty_name": "Bouachain",
+    "chain_id": "bouachain",
+    "bech32_prefix": "boua",
+    "daemon_name": "bouachain",
+    "node_home": "$HOME/.bouachain",
+    "key_algos": [
+      "secp256k1"
+    ],
+    "slip44": 118,
+    "fees": {
+      "fee_tokens": [
+        {
+          "denom": "ubouacoin",
+          "fixed_min_gas_price": 0,
+          "low_gas_price": 0.0006,
+          "average_gas_price": 0.012,
+          "high_gas_price": 0.04
+        }
+      ]
+    },
+    "staking": {
+      "staking_tokens": [
+        {
+          "denom": "ubouacoin"
+        }
+      ],
+      "lock_duration": {
+        "time": "1814400s"
+      }
+    },
+    "codebase": {
+      "git_repo": "https://github.com/Bouachain/bouachain.git",
+      "recommended_version": "v1.0.0",
+      "compatible_versions": [
+        "v0.9.0"
+      ],
+      "genesis": {
+        "name": "v1",
+        "genesis_url": "https://github.com/Bouachain/bouachain/raw/master/genesis.json"
+      },
+     "versions": [
+      {
+        "name": "v1.0.0",
+        "recommended_version": "v1.0.0",
+        "compatible_versions": [
+        "v0.9.0"
+      ],
+        "sdk": {
+          "type": "cosmos",
+          "repo": "github.com/rollchains/cosmos-sdk",
+          "version": "v0.50.8",
+          "tag": "v0.50.8"
+        }
+      }
+    ],
```

Comment:
Schema is invalid. I think it might be that you're not closing the 'codebase' object.